commit 689bf4159f668510d1c5b45daa9034f99cd47738
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-03-11T17:35:00-07:00 (7 months ago)
    
    joyent/node-triton#254 Fix Markdown syntax in README
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/README.md b/README.md
index 0fe770a..e201640 100644
--- a/README.md
+++ b/README.md
@@ -328,7 +328,7 @@ are in "etc/defaults.json" and can be overriden for the CLI in
 | Name | Description |
 | ---- | ----------- |
 | profile | The name of the triton profile to use. The default with the CLI is "env", i.e. take config from `SDC_*` envvars. |
-| cacheDir | The path (relative to the config dir, "~/.triton") where cache data is stored. The default is "cache", i.e. the `triton` CLI caches at "~/.triton/cache". |
+| cacheDir | The path (relative to the config dir, "\~/.triton") where cache data is stored. The default is "cache", i.e. the `triton` CLI caches at "\~/.triton/cache". |
 
 
 ## node-triton differences with node-smartdc
